gprofiler/log.py

```diff
@@ -199,6 +200,10 @@ def initial_root_logger_setup(
     logger_adapter = get_logger_adapter("gprofiler")
     logger_adapter.setLevel(logging.DEBUG)
 
+    # Patch formatTime to be GMT (UTC) for all formatters,
```

Comment:
If possible easily, I'd do it on the particular formatter object and not on the class. Such global will make it harder if we ever try to combine gProfiler's code into other projects.

Reply:
Done